packages/amplication-client/src/Project/WorkspaceSelectorNew.tsx

```diff
@@ -0,0 +1,157 @@
+import "./WorkspaceSelectorNew.scss";
+import React, { useCallback, useEffect, useState } from "react";
+import { gql, useQuery, useMutation, useApolloClient } from "@apollo/client";
+import classNames from "classnames";
+import { useHistory } from "react-router-dom";
+import { Button, EnumButtonStyle } from "../Components/Button";
+import { setToken } from "../authentication/authentication";
+import {
+  CircleBadge,
+  Dialog,
+  CircularProgress,
+} from "@amplication/design-system";
+import * as models from "../models";
+import WorkspaceSelectorList from "../Workspaces/WorkspaceSelectorList";
+import NewWorkspace from "../Workspaces/NewWorkspace";
+import "./WorkspaceSelectorNew";
+
+type TData = {
+  currentWorkspace: models.Workspace;
+};
+
+type TSetData = {
+  setCurrentWorkspace: {
+    token: string;
+  };
+};
+
+export const COLOR = "#A787FF";
+const CLASS_NAME = "workspaces-selector-new";
+
+const WorkspaceSelectorNew = () => {
```

Comment:
why are we using "new" in the name?

Reply:
Because I wasn't sure about it. We already have WorkspaceSerlector component in the Workspace folder. I assume we can replace it, but I didn't want to remove the original one before the review